src/services/data/DataService.test.tsx

```diff
@@ -4,7 +4,9 @@ describe("Service: DataService", () => {
   describe("#postMultipartRequest", () => {
     describe("during success", () => {
       let DataServiceSpy = jest.spyOn(DataService, "postMultipartRequest");
-      it("should call the endpoint and return a 204", () => {});
+      it("should call the endpoint and return a 200", () => {});
+      expect(true);
+      expect(DataServiceSpy).toBeTruthy(); // This does nothing.
```

Comment:
This test looks incomplete,

Reply:
Yep. It's a placeholder for now, basically.